### PR TITLE
Awesome books with class

### PR DESCRIPTION
**Pull request by Newtayo**
## Milestone

- [ ] Converted awesome books page to single page application.
- [ ] Nav Links have been added to the single page application to aid easy navigation of the pages.
- [ ] Passed all linter checks

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,5 +1,37 @@
 /* eslint-disable max-classes-per-file */
```
Comment (Ghiftee) [inline script.js]: - [ ] Let's try to not disable linters as they help us write better and cleaner code

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @Newtayo and @yodit93 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ App runs smoothly
✔️ Design matches wireframe
✔️ No linter errors

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- I suggest you add the name of the second contributor to the project to your README. This would ensure both parties get credit for their work. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Newtayo** [comment]
- [ ] Refactored the code to the use of just a class.
- [ ] Passed all linter checks
- [ ] Eslink disabled command removed from the code

File: index.html [hunk 2]
```diff
@@ -9,20 +9,44 @@
...
+    </nav>
+    <div class="date"></div>
+    <h1 class="heading">All Awesome books</h1>
+    <div class="container"> 
+    </div>
+  
+    <div class="new-data">
+      <h2>Add a New Book</h2>
+      <input type="text" class="new-title" placeholder="Title">
+      <input type="text" class="new-author" placeholder="Author">
```
Comment (AmaduKamara) [inline index.html:28]: [ Optional Changes ]

- [ ] What if you could clear the input fields when a book is added so the user doesn't have to manually clear them in order to add a new book?

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @Newtayo,

Good job working on the changes asked by the previous reviewer 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ All sections in the navigation bar work fine as expected 👍🏽 
✅ Adding and removing a book works fine 👍🏽 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**